test/performance/odp_ipsec.c

```diff
@@ -139,6 +139,13 @@ typedef struct {
 	 * Specified through -u argument.
 	 */
 	int ah;
+
+	/*
+	 * Burst size.
+	 * Prepare and submit as many packets for IPsec processing in each
+	 * iteration of the loop.
+	 */
+	int burst_sz;
```

Comment:
Maybe burst_size would be better name than burst_sz in all places where burst_sz is used?

Reply:
Agreed. Will make this change in v2.